src/main/java/core/basesyntax/service/ReportGenerator.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ReportGenerator {
```

Comment:
```suggestion
public class ReportGenerator {
```
The same problems here. Please fix it in all places